### DISCUSSION TITLE
How to "change the storage type to Sqlite"

### DISCUSSION THREAD

**Discussion by eastnoob** [Tunny-gh/Tunny_in_Open_Sea#187] [General]
How to modify the "storage type" because when I try to execute A, the system prompts me to “change the storage type to Sqlite", but I cannot find relative setting in "setting" menu.

![image](https://github.com/hrntsm/Tunny/assets/61303573/3a29c7ba-4a90-4187-aa99-f8f686d288fe)

**Comment by hrntsm** [comment]
Hi @eastnoob ,Thanks for your interest in Tunny.
Please check following discussion
https://github.com/hrntsm/Tunny/discussions/183#discussioncomment-6586822

**Reply by eastnoob** [reply]
Thanks！
